lib/api.js

```diff
@@ -164,6 +165,11 @@ PlexAPI.prototype._request = function _request(options) {
 
         resolveValue = body;
 
+        if (response.statusCode === 403 && !options.overrideToken && self.authToken) {
```

Comment:
Mentioned this scenario [here](https://github.com/phillipj/node-plex-api-credentials/pull/33#discussion_r65181106). 403 is returned when you have a valid token but you don't have access to the resource with that token.

Alternatively I suppose the code could be throw a different error and the user of this client could update the token and try again but if this is a common case and we have both tokens I figured why not just try the same operation again with the auth token?